dist/scss/wds-components/inputs/_input.scss

```diff
@@ -63,6 +63,14 @@
 		color: $wds-color-blue-gray;
 	}
 
+	.is-empty > &__field ~ &__label {
+		overflow: hidden;
+		height: 1rem;
+		text-overflow: ellipsis;
```

Comment:
I'm not totally sure this is a good approach. We will truncate the text in the hint which kind of defeats its purpose.

Reply:
this selector is already defined in line 56. let's move these styles there